### PR TITLE
[DC-2436] Update `bq.build_and_copy_contents` to copy tables without pre-defined schema in resource_files/schemas.

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

**Review by hiro-mishima — APPROVED** [review]
It looks good to me! I also confirmed this change works by generating Fitbit CT DEID creation.
Thanks, @ksdkalluri!

File: data_steward/utils/bq.py [hunk 1]
```diff
@@ -676,28 +676,38 @@ def build_and_copy_contents(client, src_dataset, dest_dataset):
 
     for table_item in table_list:
         # create empty schemaed tablle with client object
-        schema_list = get_table_schema(table_item.table_id)
+        try:
+            schema_list = get_table_schema(table_item.table_id)
+        except RuntimeError as re:
+            schema_list = None
```
Comment (lrwb-aou) [inline data_steward/utils/bq.py:682]: I would ask for a logging statement added here so we know what is happening.

```suggestion
            schema_list = None
            LOGGER.warning(f"No schema available for {table_item.table_id}.  This function will "
                           f"copy table contents without specifying a schema.")
```

**Review by lrwb-aou — CHANGES_REQUESTED** [review]
I think the PR should work, but would appreciate the logging.  Also, would it be better for this to fail to force the usage of schema files?

File: data_steward/utils/bq.py (hunk 1 above)
Comment (ksdkalluri) [reply data_steward/utils/bq.py:682]: @lrwb-aou Added the log.

**Review by lrwb-aou — APPROVED** [review]
Thank you!  Looks good.

**Pull request merged**